src/main/java/jenkinsci/plugins/influxdb/generators/ChangeLogPointGenerator.java

```diff
@@ -0,0 +1,100 @@
+package jenkinsci.plugins.influxdb.generators;
+
+import java.util.Collection;
+import java.util.Iterator;
+
+import org.influxdb.dto.Point;
+
+import hudson.model.AbstractBuild;
+import hudson.model.Run;
+import hudson.scm.ChangeLogSet;
+import jenkinsci.plugins.influxdb.renderer.MeasurementRenderer;
+
+public class ChangeLogPointGenerator extends AbstractPointGenerator {
+    
+	public static final String BUILD_DISPLAY_NAME = "display_name";
+
+	private final Run<?, ?> build;
+	private final String customPrefix;
+	
+	private StringBuilder affectedPaths;
+
+	private StringBuilder messages;
+
+	private StringBuilder culprits;
+
+	private int commitCount = 0;
+
+	public ChangeLogPointGenerator(MeasurementRenderer<Run<?, ?>> projectNameRenderer, String customPrefix,
+			Run<?, ?> build) {
+		super(projectNameRenderer);
+		this.build = build;
+		this.customPrefix = customPrefix;
+	}
+
+	public boolean hasReport() {
+		if (build instanceof AbstractBuild) {
+			getChangeLog(build);
+			return true;
+		}
+		return false;
+	}
+
+	public Point[] generate() {
+		
+		Point.Builder point = buildPoint(measurementName("changelog_data"), customPrefix, build);
+
+		point.field(BUILD_DISPLAY_NAME, build.getDisplayName())
+				.field("commit_messages", this.getMessages())
```

Comment:
Due to influx-java version update, these function calls are now deprecated. You should use .addField() instead of .field().